priam/src/test/java/com/netflix/priam/identity/DoubleRingTest.java

```diff
@@ -15,19 +15,13 @@
  *
  */
 
-package com.netflix.priam.backup.identity;
+package com.netflix.priam.identity;
```

Comment:
I moved these tests to same package as the classes they test, so they have access to the now-package-private instance properties.